src/ui/utils/workspace.ts

```diff
@@ -40,7 +38,7 @@ export function subscriptionEndsIn(workspace: Workspace, date?: Date | number) {
 export function subscriptionExpired(workspace: Workspace, date?: Date) {
   const expiresIn = subscriptionEndsIn(workspace, date);
 
-  return expiresIn <= 0;
+  return expiresIn < 0;
```

Comment:
Haven't wrapped my head around this fully. We're looking for a negative `expiresIn` value to indicate that it's expired?

Reply:
Oh we count calendar days got it got it.